### PR TITLE
Add custom dark/light mode SVG support

### PR DESCRIPTION
**Pull request by Malyuk-A**
After several hours of tinkering I conclude that the native dark-light toggle from Doks/Hugo is working - but the `inline-svg` is not compatible with "dark-mode" SVGs created via draw.io.

E.g. I created and provide only a dark mode option of an SVG but it was still rendered white.

So instead I added a new custom shortcode that does the trick. In addition you only need to provide the prefix name of the file.

E.g.:
```
Files:
- your_svg.drawio
- your_svg_dark.svg
- your_svg_light.svg

Embedded in the md files like this:

{{<svg "your_svg">}}
```

One has to place these files in the `static/svgs/` folder

Demo:

https://github.com/user-attachments/assets/566a3e3e-99e9-4805-865a-f9624ad4d3ae

https://github.com/user-attachments/assets/9094d84d-25c0-4284-8b56-7d5062bf8f59

### CODE REVIEW

File: static/svgs/mqtt/MQTT-Arch.svg
Comment (Mjaethers) [inline static/svgs/mqtt/MQTT-Arch.svg:1]: Was keeping the previously light svg deliberate? Perhaps keeping a *.draw.io file makes it clear that this file can be imported into drawio, even if importing a drawio *.svg is not a problem.

**Review by Mjaethers — COMMENTED** [review]
Really really nice. Let's merge this ASAP so that we can start using it!

File: assets/svgs/mqtt/MQTT-Arch.drawio
Comment (giobart) [inline assets/svgs/mqtt/MQTT-Arch.drawio:1]: can we keep the .drawio files as well?

**Review by giobart — CHANGES_REQUESTED** [review]

File: static/svgs/mqtt/MQTT-Arch.svg
Comment (Malyuk-A) [reply static/svgs/mqtt/MQTT-Arch.svg:1]: My bad - this should have been the .drawio file - maybe "file name typo" from my side - As described in the PR description I of course, want to keep the drawio file :P

File: assets/svgs/mqtt/MQTT-Arch.drawio
Comment (Malyuk-A) [reply assets/svgs/mqtt/MQTT-Arch.drawio:1]: My bad - this should have been the .drawio file - maybe "file name typo" from my side - As described in the PR description, I of course, want to keep the drawio file :P

**Review by giobart — APPROVED** [review]

**Comment by giobart** [comment]
Awesome! Feel free to merge, green light on my side

**Pull request merged**